lang/cs/Org.Apache.REEF.Common/Telemetry/MetricsService.cs

```diff
@@ -113,5 +113,18 @@ public void OnCompleted()
         public void OnError(Exception error)
         {
         }
+
+        /// <summary>
+        /// Observer of IDriverMetrics.
+        /// When Driver metrics data is changed, this method will be called
+        /// </summary>
+        /// <param name="value"></param>
+        public void OnNext(IDriverMetrics value)
+        {
+            var set = new HashSet<KeyValuePair<string, string>>();
```

Comment:
The data is for Sink which only need a set. It will simply dump the data into a file in the real implementation. So far the design is to use string to represent value no matter what the value type is as string is generic and the data will be saved into a file in string format.